content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -219,9 +219,9 @@ You can give feedback in the [discussion forum][1], or send an e-mail to [team-i
 [6]: /tracing/error_tracking/
 [7]: https://www.jetbrains.com/lp/toolbox/
 [8]: /security/application_security/vulnerability_management/
-[9]: /continuous_integration/guides/flaky_test_management/
+[9]: /tests/guides/flaky_test_management/
 [10]: /watchdog/insights
 [11]: mailto:team-ide-integration@datadoghq.com
 [12]: /continuous_integration/explorer/?tab=testruns
-[13]: /static_analysis/?tab=githubactions
-[14]: /static_analysis/rules/#python-rules
\ No newline at end of file
+[13]: /code_analysis/static_analysis/?tab=githubactions
+[14]: /code_analysis/static_analysis_rules/#python-rules
```

Comment:
@alai97 also, this should probably be https://docs-staging.datadoghq.com/alai97/small-link-cleanup-before-merging/code_analysis/static_analysis_rules/?languages=Python

ending in `?languages=Python` instead of `#python-rules`

Reply:
```suggestion
[14]: /code_analysis/static_analysis_rules/?languages=Python
```